src/wp-includes/blocks.php

```diff
@@ -181,7 +181,7 @@ function register_block_script_handle( $metadata, $field_name, $index = 0 ) {
 	$script_uri       = get_block_asset_url( $script_path_norm );
 
 	$script_args = array();
-	if ( 'viewScript' === $field_name ) {
+	if ( 'viewScript' === $field_name && $script_uri ) {
```

Comment:
This works too. But it's just that it's not ideal from a code coverage perspective, since the in normal execution the condition would be truthy. It's just that during PHPUnit tests that it is false. So this is essentially adding a condition like `! $is_running_unit_tests`

Reply:
@westonruter This is a valid case to catch outside of unit tests. If you pass an empty thing as a path to script, this should result false. For example, if you using a unbuilt version of plugin for development this might happen.

Reply:
Does this solve https://core.trac.wordpress.org/ticket/59475?

Reply:
@swissspidy yes